libs/state/tsconfig.spec.json

```diff
@@ -7,5 +7,11 @@
     "target": "es2016"
   },
   "files": ["src/test-setup.ts"],
-  "include": ["jest.config.ts", "src/**/*.test.ts", "src/**/*.spec.ts", "src/**/*.d.ts"]
+  "include": [
+    "jest.config.ts",
+    "src/**/*.test.ts",
+    "src/**/*.spec.ts",
+    "src/**/*.d.ts",
+    "../services/src/lib/tissue-library/tissue-library.service.spec.ts"
```

Comment:
Why are you adding this?

Reply:
I am not sure how it went there, I will remove it.